enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -1063,6 +1063,9 @@ def redeem(self, lms_user_id, content_key, all_transactions, metadata=None, **kw
                 requested_price_cents = kwargs.get('requested_price_cents')
                 if requested_price_cents is not None:
                     creation_payload['requested_price_cents'] = requested_price_cents
+                external_fulfillment_reference_id = kwargs.get('external_fulfillment_reference_id')
+                if external_fulfillment_reference_id is not None:
+                    creation_payload['external_fulfillment_reference_id'] = external_fulfillment_reference_id
```

Comment:
I think this was dead code leftover from a refactor. The approach actually used was to pass the UUID via transaction metadata, not as a net-new create_subsidy_transaction() kwarg. If you follow this kwarg into the [client function code](https://github.com/openedx/edx-enterprise-subsidy-client/blob/8a6579b96abf4e927181fc17b9b1fb8b1aa56005/edx_enterprise_subsidy_client/client.py#L231-L239), it actually doesn't have a matching kwarg, and would throw an "unexpected keyword argument" error if actually passed.

Reply:
fix -> https://github.com/openedx/enterprise-access/pull/834